pkgs/development/python-modules/labgrid/default.nix

```diff
@@ -32,13 +35,17 @@ buildPythonPackage rec {
     sha256 = "sha256-yhlBqqCLOt6liw4iv8itG6E4QfIa7cW76QJqefUM5dw=";
   };
 
-  patches = [
-    # Pyserial within Nixpkgs already includes the necessary fix, remove the
-    # pyserial version check from labgrid.
-    ./0001-serialdriver-remove-pyserial-version-check.patch
+  # pythonRelaxDepsHook is used to relax all dependencies. Can be removed with
+  # the next labgrid release, master already switched to more relaxed
+  # dependencies.
```

Comment:
`pythonRelaxDepsHook` currently does nothing, since `pythonRelaxDeps` is not set.

Reply:
Indeed, removing it also lets the package build. I looked into the docs and code, but couldn't find out why I no longer need to remove the hard dependencies inside the `pyproject.toml`. If someone knows why please post :smile: Pushed a version without `pythonRelaxDepsHook`.